ServerSide/yoty/Data/Entities/SellerOffersEntity.cs

```diff
@@ -4,25 +4,25 @@ namespace YOTY.Service.Data.Entities
 {
     public class SellerOfferEntity
```

Comment:
rename it supplierPropsal ... 
or keep it like that but change supplier to seller and offer to proposal here and every where, lets be consistent we our naming decisions